packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -78,6 +80,18 @@ class FlutterPlugin implements Plugin<Project> {
                     matchingFallbacks = ['debug', 'release']
                 }
             }
+            dynamicProfile {
+                initWith debug
+                if (it.hasProperty('matchingFallbacks')) {
+                    matchingFallbacks = ['debug', 'release']
```

Comment:
Is setting `matchingFallbacks` like this for these dynamic configurations as intended?

Reply:
I suspect this setting is inconsequential. It only matters if one of flutter's dependency modules doesn't define given build configuration, and I don't see any sub-modules.

Reply:
cc @mravn-google in case he knows